src/NetMock/Rest/RestMock.cs

```diff
@@ -133,6 +135,11 @@ public void Activate()
 			if (ServiceMock.ActivationStrategy == ActivationStrategy.Manual)
 				ParseRequestDefinitions();
 
+			if (Certificate != null && InstallCertificate)
+			{
+				CertificateUtil.AddCertificate(Certificate);
```

Comment:
Return a value indicating whether or not the certificate was installed, and use that below to only uninstall the certificate if we previously installed it.

Reply:
Only removing the certificate if it has been previously installed by the Restmock.